spec/models/services/tumblr_spec.rb

```diff
@@ -11,7 +11,9 @@
 
   describe '#post' do
     it 'posts a status message to tumblr and saves the returned ids' do
-      response = double(body: '{"response": {"user": {"blogs": [{"url": "http://foo.tumblr.com"}]}}}')
+      response = double(body: '{"response": {"user": {"blogs":
+       [{"primary": true, "url": "http://foo.tumblr.com"},
+        {"primary": false, "url": "http://bar.tumblr.com"}]}}}')
```

Comment:
Let's make sure it's not just picking the first one and order the primary behind the non-primary one.